docs/user_guide/remote_processing.rst

```diff
@@ -7,98 +7,51 @@ The SiliconCompiler project supports a remote processing model that leverages th
-
-.. list-table::
-   :widths: 10 10
-   :header-rows: 1
-
-   * - Keypath
-     - Restrictions
-
-   * - ``design``
-     - Must only contain alphanumeric characters or underscores
-   * - ``jobname``
-     - Must only contain alphanumeric characters or underscores
-   * - ``dir``
-     - Must only contain alphanumeric characters or underscores
-   * - ``constraint``
-     - Only allowed for fpgaflow
-   * - ``target``
-     - Must be one of ``asicflow_freepdk45``, ``asicflow_skywater130``, or ``fpgaflow_ice40up5k-sg48``
-   * - ``library, ...``
-     - Allowed if key isn't the name of a built-in PDK standard cell library
-   * - ``source``
-     - None
-   * - ``testbench``
-     - None
-   * - ``waveform``
-     - None
-   * - ``clock, ...``
-     - None
-   * - ``supply, ...``
-     - None
-   * - ``param, ...``
-     - None
-   * - ``define``
-     - None
-   * - ``ydir``
-     - None
-   * - ``idir``
-     - None
-   * - ``vlib``
-     - None
-   * - ``libext``
-     - None
-   * - ``cmdfile``
-     - None
-   * - ``activityfactor``
-     - None
-   * - ``read, spef, ...``
-     - None
-   * - ``read, sdf, ...``
-     - None
-   * - ``read, saif, ...``
-     - None
-   * - ``read, gds, ...``
-     - None
-   * - ``read, def, ...``
-     - None
-   * - ``read, netlist, ...``
-     - None
-   * - ``read, pcf, ...``
-     - None
-   * - ``read, vcd, ...``
-     - None
-   * - ``asic, ...``
-     - None
-   * - ``metric, ...``
-     - None
-   * - ``fpga, ...``
-     - None
-   * - ``flowarg, ...``
-     - None
-   * - ``techarg, ...``
-     - None
-   * - ``clean``
-     - None
-   * - ``loglevel``
-     - None
-   * - ``novercheck``
-     - None
-   * - ``checkonly``
-     - None
-   * - ``eda, ..., variable, ...``
-     - None
+Remote processing is also supported from the Python interface when the :keypath:`option, remote` parameter is set to 'True'. To get a feel for the Python interface, you can try running some of our `example designs <https://github.com/siliconcompiler/siliconcompiler/tree/main/examples/>`_ with this flag enabled::
+
+  chip.set('option', 'remote', True)
+
+Troubleshooting
+---------------
+
+Our public beta servers do not prune or pre-process Schema parameters, in order to make the remote processing environment as close to a local environment as possible. The jobs will be run in isolated environments with limited communication interfaces, however, so they jobs may not be able to perform network calls or meaningful filesystem access when they are run on our cloud beta servers.
+
+Any changes that you make to SiliconCompiler's built-in tool setup scripts on your local machine will not be reflected in jobs which are run on a remote server. Likewise, any changes that you make to the built-in open-source PDKs and standard cell libraries will not be sent to the remote servers.
```

Comment:
Maybe add a link to the contributions page and encourage them to contribute the changes